src/SDK/MonitoringApplianceEmulator/ViewModels/AlertDetailsControlViewModel.cs

```diff
@@ -143,5 +148,115 @@ private set
         public CommandHandler CloseControlCommand { get; }
 
         #endregion
+
+        private async Task<List<DisplayableAlertProperty>> ComposeArmProperties(AzureResourceManagerRequestAlertProperty armProperty, IAzureResourceManagerClient armClient)
+        {
+            var url = armProperty.AzureResourceManagerRequestUri;
```

Comment:
>url [](start = 16, length = 3)

consider removing this variable - might be clearer what's the meaning when you use it